content/guides/code-splitting-import.md

```diff
@@ -144,6 +144,28 @@ module.exports = {
 };
 ```
 
+## `import` imports the entire module namespace
+
+Consider the following two examples:
```

Comment:
Haven't gotten a response here yet: https://github.com/tc39/proposal-dynamic-import/issues/21#issuecomment-294293874

Do you know where to find the actual spec?